fake-snippets-api/lib/db/schema.ts

```diff
@@ -100,6 +100,39 @@ export const orderFileSchema = z.object({
 })
 export type OrderFile = z.infer<typeof orderFileSchema>
 
+export const orderQuoteComponentSchema = z.object({
+  component_id: z.string(),
+  name: z.string(),
+  manufacturer_part_number: z.string().nullable(),
+  jlcpcb_part_number: z.string().nullable(),
+  quantity: z.number(),
+  unit_price: z.number(),
+  total_price: z.number(),
+  type: z.string(), // e.g., "resistor", "capacitor", etc.
+  available: z.boolean().default(true),
+})
+export type OrderQuoteComponent = z.infer<typeof orderQuoteComponentSchema>
+
+export const orderQuoteSchema = z.object({
+  order_quote_id: z.string(),
+  account_id: z.string().nullable(),
+  package_release_id: z.string().nullable(),
+  is_completed: z.boolean().default(false),
+  is_processing: z.boolean().default(true),
+  error: errorSchema.nullable(),
+  has_error: z.boolean().default(false),
+  created_at: z.string(),
+  updated_at: z.string(),
+  completed_at: z.string().nullable(),
+  bom_rows: z.string(),
```

Comment:
I dont know if we need to store a string here

Reply:
So, only the part numbers from the bom values?

Reply:
We wouldnt use bom_rows in the ui i think, if we do they should probably be parsed